### ISSUE TITLE
[Bug] 新创建的评测任务 dimension 字段为 null

### ISSUE THREAD

**Issue by chenxibj** [chenxibj/ai-hardware-verification-platform#151]
## 问题描述

通过预设方案自动生成的评测任务中，所有任务的 `dimension` 字段都为 null。

## 复现步骤

1. 创建芯片并用任意预设创建评测计划
2. GET /api/tasks?planId={新计划ID}
3. 检查返回的任务列表

## 实际结果

```json
{"id": 631, "testItem": "MatMul", "dimension": null}
{"id": 632, "testItem": "Conv2D", "dimension": null}
{"id": 633, "testItem": "Softmax", "dimension": null}
```

所有任务的 dimension 都是 null。

## 预期结果

dimension 应根据 testItem 自动分类，例如：
- MatMul, Conv2D → 计算性能
- Softmax, LayerNorm, BatchNorm → 归一化
- ReLU, GELU, SiLU → 数学函数
- Attention, ScaledDotProduct → 注意力机制
- MLP-Medium → 其他/端到端

## 影响

缺少 dimension 分类会影响前端的维度评分展示和雷达图生成。

## 关联

- 回归测试新发现
- 标签: bug, P1
